src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -31,6 +37,26 @@ const formConfig = {
           path: 'form-page',
           title: 'First Page',
           uiSchema: {
+            name: _.merge(autoSuggestUiSchema(
```

Comment:
This uses the existing `autoSuggestUiSchema` and replaces the `ui:field` that's used with the new version